views/shop/index.hbs

```diff
@@ -1,28 +1,29 @@
-<div class="row">
-  <div class="col-sm-6 col-md-4 col-md-offset-4 col-sm-offset-3 mx-auto">
-    <div id="success" class="alert alert-success" {{#if noMessages}}hidden{{/if}}>
-      {{ successMsg }}
-    </div>
-  </div>
-</div>
-{{# each services }}
-<div class="row">
-  {{# each this }}
-  <div class="col-sm-6 col-md-4">
-    <div class="card-deck">
-      <div class="card">
-      <img src="{{ this.imagePath }}" alt="deck" class="img-responsive">
-        <div class="card-body">
-        <h3>{{ this.title }}</h3>
-        <p class="description">{{ this.description }}</p>
-          <div class="clearfix">
-            <div class="price pull-left">${{ this.price }}</div>
-            <a href="/add-to-cart/{{this._id}}" class="btn btn-success pull-right" role="button">Add to Shopping Cart</a>
+<p>
+<div class="container">
+
+  {{# each services }}
+
+  <div class="card-deck mb-3 ">
+
+    {{# each this }}
+
+
+        <div class="card mb-4 box-shadow">
+          <div class="embed-responsive embed-responsive-16by9">
+            <img alt="deck" class="card-img-top embed-responsive-item" src="{{ this.imagePath }}" />
           </div>
+          <div class="card-body d-flex flex-column">
+            <h1 class="card-title pricing-card-title text-center">${{ this.price }}</h1>
+            <ul class="list-unstyled mt-3 mb-4">
+              <li>{{ this.description }}</li>
+            </ul>
+
+            <a href="/add-to-cart/{{this._id}}" class="btn mt-auto btn-success " role="button">Add to Shopping Cart</a>
+
+
```

Comment:
More blank lines? Also were did clearfix go.

Without the clearfix the wrapping div would not span around the buttons which would cause a broken layout.